<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomList</class>
 <widget class="QWidget" name="AtomList">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="displayButton">
     <property name="text">
      <string>Atoms</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="atomFmtContainer">
        <item>
         <widget class="QLabel" name="atomFmtLabel">
          <property name="text">
           <string>Coordinate format:</string>
          </property>
          <property name="buddy">
           <cstring>atomFmtBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="atomFmtBox">
          <item>
           <property name="text">
            <string>Crystal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ångström</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bohr</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="atomFmtButton">
          <property name="text">
           <string>Set Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="atomHelpButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="atomTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
